BrainPortal/app/models/tool_config.rb

```diff
@@ -352,7 +375,7 @@ def data_providers_with_overlays
     return @_data_providers_with_overlays_ if @_data_providers_with_overlays_
     specs = self.singularity_overlays_specs.presence
     return [] if specs.blank?
-    specs = specs.split(/[\s,]+/).map(&:presence).compact
+    specs = specs.split(/([\s,]|#.*$)+/).map(&:presence).compact
```

Comment:
I think this parsing, which duplicates what we do above, should be extracted in a method to make it more formal.

I suggest a method that returns an array of pairs, `[ [ scheme, value ], [scheme, value] ... ]`

Reply:
ok